projects/ccd-case-ui-toolkit/src/lib/shared/services/form/field-type-sanitiser.ts

```diff
@@ -50,7 +51,21 @@ export class FieldTypeSanitiser {
           }
           break;
       }
+    });
+  }
 
+  private ensureDynamicMultiSelectListPopulated(caseFields: CaseField[]): CaseField[] {
+    return caseFields.map((field) => {
+      if (field.field_type.type !== 'Complex') {
+        return field;
+      }
+      const complexFieldsUpdated = field.field_type.complex_fields.map((complexField) => {
+        if (complexField.field_type.type === FieldTypeSanitiser.FIELD_TYPE_DYNAMIC_MULTISELECT_LIST && complexField.display_context !== 'HIDDEN' && field._value && field._value[complexField.id]) {
+          return { ...complexField, list_items: field._value[complexField.id].list_items };
+        }
+        return complexField;
```

Comment:
Minor comment: The return complexField here made it a bit confusing to read on initial glance. I'd suggest a ternary operator but since the if statement is quite long it's up to you. Will approve